src/specBuilder/area/areaUtils.test.ts

```diff
@@ -18,7 +24,7 @@ describe('getAreaMark', () => {
 		expect(
 			getAreaMark({
 				name: 'area0',
-				color: 'color',
```

Comment:
There are a few places where we swapped 'color' with DEFAULT_COLOR. default color is 'series'. Just want to be sure that's intentional since this is intended as a refactor specifically.